### ISSUE TITLE
Loss curve on graph for neural network

### ISSUE THREAD

**Issue by PeterDunlop** [GoodVibeSquad/first_itteration#136]
- [ ] Plot neura loss curve and accuracy curve

For some reason. The confidence of the network is quite low. When predicting a number it doesn't do it with much confidence.

When implementing softmax with this line in the forwardpass:
activations[-1] = self.activation_functions[1].run(activations[-1])

It returns wayy better data. But on newmain branch there is print where it is confident in some numbers, but these numbers do not match the successes. Figure out why this is.

It also improves loss like crazy. I hope we can use this.
